### DISCUSSION TITLE
How to have two forms connect to the same EmailJS template

### DISCUSSION THREAD

**Discussion by raven-rhea** [emailjs-com/help#161] [Q&A]
Hello, I am trying to figure out how to have 2 forms on my website that both trigger the same email template. Currently when I create the second one the first one stops working. Each form has a unique id(leadMagnet, leadMagnet2), but otherwise my code is the same. I am not an expert in javascript so any help is much appreciated. Code below. . . 

<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@emailjs/browser@4/dist/email.min.js"></script>
    <script type="text/javascript">
        (function() {
            emailjs.init({
              publicKey: "0of8uh0upbouN****",
            });
        })();
    </script>
    <script type="text/javascript">
        window.onload = function() {
            document.getElementById('leadMagnet2').addEventListener('submit', function(event) {
                event.preventDefault();
                emailjs.sendForm('service_0b*****, 'lead_magnet', this)
                    .then(() => {
                        console.log('SUCCESS!');
                    }, (error) => {
                        console.log('FAILED...', error);
                    });
            });
        }
    </script>

Thanks

**Comment by xr0master** [comment]
The number of forms should not affect the work in any way.

What does it mean your first form stopped working? Are you getting an error?

**Reply by raven-rhea** [reply]
When I click submit it reloads the page as apposed to the "Thank You"  message but doesn't show an error message.

**Reply by xr0master** [reply]
You should only reload the page when the request is finished, otherwise you cancel all JS requests.
